servers/jax-rs-testextension/src/main/java/org/projectnessie/jaxrs/ext/NessieJaxRsExtension.java

```diff
@@ -159,6 +167,17 @@ public Object resolveParameter(
             + parameterContext.getTarget());
   }
 
+  @Override
+  protected URI findBaseUri(ExtensionContext extensionContext) {
+    EnvHolder env = extensionContext.getStore(NAMESPACE).get(EnvHolder.class, EnvHolder.class);
+    if (env == null) {
+      throw new ParameterResolutionException(
+          "Nessie JaxRs env. is not initialized in " + extensionContext.getUniqueId());
+    }
```

Comment:
this logic seems duplicated, instead we should probably have a `private EnvHolder getEnv()` helper method that includes this check and gets used everwhere, wdyt ?